core/src/test/java/com/linecorp/armeria/client/ClientRequestContextDelayedInitTest.java

```diff
@@ -79,19 +80,28 @@ void timeout() {
      * properly at different points of execution.
      */
     @ParameterizedTest
-    @CsvSource({ "0", "1", "2", "3" })
+    @CsvSource({ "0", "1", "2" })
```

Comment:
one `selectNow()` has been removed so the maximum number of invocation of `selectNow()` is 3.

Reply:
https://github.com/line/armeria/pull/5163/files#diff-c61e716447f997dea21163d52d7a0a59fc77c25f1cab14426875c7b109876800L80